tools/deploy_manifests.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+platform=$1
+image=$2
+
+namespace=${TEST_NAMESPACE}
```

Comment:
I think it could be nice to have `set -u` in order to make sure that the script will fail if the variables are not set properly.

Also maybe namespace can be passed to the script as a parameter, it would make it consistent with `platform` and `image`, what do you think?